squid-blockexplorer/src/balances.ts

```diff
@@ -95,23 +97,22 @@ async function saveAccounts(
         total,
         updatedAt: BigInt(block.height),
       });
-      accounts.set(id, account);
-    } else {
+      accounts.push(account);
+    } else if (!existingAccount?.extrinsics.length) {
```

Comment:
i can't think of a case where a balance is going to 0 and there hasn't been some extrinsic associated with it, can you @isSerge ? If not, I think we should do away with the delete logic altogether.

Reply:
this is definitely not a trivial case, but it is possible to set `Pays::No` to a call inside a pallet, so technically it is possible to have zero balance and have some transactions submitted

Reply:
but in that case we won't see that account in `processBalances` anyway...